### ISSUE TITLE
Backspace won't remove if input field is a number type

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noctarius/chromebackspaceonlinux#32]
```
What steps will reproduce the problem?
1. Go to a page with some input number type 
(http://www.w3schools.com/html5/tryit.asp?filename=tryhtml5_input_type_number)
2. Type something...
3. Then try to backspace it to remove

What is the expected output? What do you see instead?
Text deletion. Instead it goes back...

What version of the product are you using? On what operating system?
Version 0.52.0 ; Linux, xUbuntu 11.10



```

Original issue reported on code.google.com by `asloNon...@gmail.com` on 30 Mar 2012 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
Will be fixed the next days
```

Original comment by `noctariushtc@googlemail.com` on 27 Jul 2012 at 4:58
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Any progress on that?
I think(unless I'm wrong) this should pretty much be a matter of checking the 
event target and making sure to ignore the action if it's a input[type="text"]. 

It's a pretty minor issue, but every now and then I hit backspace while 
entering stuff in a number field and go back. If it wasn't for the fact that 
the browser remembers the state of all fields when I hit forward I would've 
been quite sad every time that happens. 
```

Original comment by `nikolov....@gmail.com` on 26 Feb 2014 at 10:39

**Comment by noctarius** [comment]
Fixed in 1.0.1
